### PR TITLE
Update token v0.127 to v0.132

### PR DESCRIPTION
**Pull request by QuncCccccc**
* Updated the Material Design tokens to v0.132.
* New token sets: Progress indicators(circular and linear), tab, and bottom sheet.
* Updated the private class name for ChoiceChip from `_FilterChipDefaultsM3` to `_ChoiceChipDefaultsM3`

### CODE REVIEW

**Review by HansMuller — APPROVED** [review]
LGTM

File: packages/flutter/lib/src/material/choice_chip.dart [hunk 1]
```diff
@@ -148,7 +148,7 @@ class ChoiceChip extends StatelessWidget
     assert(debugCheckHasMaterial(context));
     final ChipThemeData chipTheme = ChipTheme.of(context);
     final ChipThemeData? defaults = Theme.of(context).useMaterial3
-      ? _FilterChipDefaultsM3(context, isEnabled, selected)
+      ? _ChoiceChipDefaultsM3(context, isEnabled, selected)
```
Comment (darrenaustin) [inline packages/flutter/lib/src/material/choice_chip.dart:151]: Thanks for doing this, I just caught this myself working on another Chip issue. Not sure how this got messed up.

**Review by darrenaustin — APPROVED** [review]
LGTM. Awesome. Thx for doing this.

File: packages/flutter/lib/src/material/choice_chip.dart (hunk 1 above)
Comment (QuncCccccc) [reply packages/flutter/lib/src/material/choice_chip.dart:151]: No problem!

**Pull request merged**